File: DS-Algo-with-Java/Lecture-9/Triangle.java
class Tri {
    int a, b, c;

    public double getArea() {
        double s = (a + b + c) / 2.0;
        return Math.sqrt((s * (s - a) * (s - b) * (s - c)));
    }

    public double getPerimeter() {
        return (a + b + c) / 2.0;
    }
}

public class Triangle {
    public static void main(String[] args){
        Tri obj = new Tri();
        obj.a = 2;
        obj.b = 5;
        obj.c = 6;
        System.out.println(obj.getArea());
        System.out.println(obj.getPerimeter());
    }
}

File: DS-Algo-with-Python/Lecture-9/Triangle.py
import math

class Triangle:
    a = 0
    b = 0
    c = 0

    def getArea(self):
        s = (self.a + self.b + self.c) / 2

        return math.sqrt(s * (s - self.a) * (s - self.b) * (s - self.c))
    
    def getPerimeter(self):

        return (self.a + self.b + self.c) / 2


obj = Triangle()
obj.a = 2
obj.b = 5
obj.c = 6

print(obj.getArea())
print(obj.getPerimeter())
